MovingBlocks/Terasology
# Fixed Issue #2921 and Additional Translation [Only Hindi Translation]

### Contains
 
Fixed issue #2921 and added some additional translation regarding "Telemetry" and "Storage Services".

### How to test

1. Start the game
2. Change the language to Hindi
3. See the changes :+1:
